src/components/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { FooterProps } from '../types/Props';
+import { FilterStates } from '../types/enums';
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  filter,
+  setFilter,
+  onClearCompleted,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodos} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {Object.values(FilterStates).map(state => (
+          <a
+            key={state}
+            href={`#/${state}`}
+            className={`filter__link ${filter === state ? 'selected' : ''}`}
```

Comment:
use [classnames](https://www.npmjs.com/package/classnames) library for it. In all such cases